### PR TITLE
Integrational tests

### PR DESCRIPTION
**Pull request by Mov305**
### HI there!

1- Installed capybara
2- Fixed the n + 1 problem
2- Added the international tests
3- Checked The linters
4- Checked all the requirements
image

### CODE REVIEW

File: spec/views/users/user_show_spec.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    click_link(@user1.name)
+    expect(page).to have_content(@user1.posts.count)
+  end
+
+  it 'displays the user posts' do
+    visit users_path
+    click_link(@user1.name)
+    expect(page).to have_content(@post1.title)
+  end
+
```
Comment (mikethreels) [inline spec/views/users/user_show_spec.rb:46]: - [ ] you should check in this case if the first 3 posts are visible.
- [ ] you should also check if there is a button visible that lets me view all of a user's posts.
- [ ] Tests that redirect to the post's show page when you click on a user's post.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Mov, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/views/users/user_show_spec.rb (hunk 1 above)
Comment (Mov305) [reply spec/views/users/user_show_spec.rb:46]: It didn't mention three posts in the requirement I so I only added two

Comment (rloterh) [reply spec/views/users/user_show_spec.rb:46]: resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Mov,  

Your project is complete! 🏁 Great job! :gem: :gem: There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 👍
- Good job making recommended changes!

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
- Kindly include appropriate instructions in your ReadMe on how to run the _Rspec_ specs for this app, under the _Getting Started_ section. 
Implementing the above change(s) will make your ReadMe/ repository stand out even better! 😉

Cheers..Happy coding! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**